quickwit/quickwit-ingest/src/ingest_v2/ingester.rs

```diff
@@ -968,6 +969,19 @@ impl IngesterService for Ingester {
         &mut self,
         persist_request: PersistRequest,
     ) -> IngestV2Result<PersistResponse> {
+        // If the request is local, the amount of memory it occupies is already
+        // accounted for in the router.
+        let request_size_bytes = persist_request
+            .subrequests
+            .iter()
+            .flat_map(|subrequest| match &subrequest.doc_batch {
+                Some(doc_batch) if doc_batch.doc_buffer.is_unique() => Some(doc_batch.num_bytes()),
```

Comment:
cool so you found a workaround.

Reply:
I asked on the tokio Discord server when they will release the new version of `bytes` and they did the same day.